src/js_native_api_v8.h

```diff
@@ -140,14 +140,22 @@ napi_status napi_set_last_error(napi_env env, napi_status error_code,
   (!try_catch.HasCaught() ? napi_ok \
                          : napi_set_last_error((env), napi_pending_exception))
 
-#define THROW_RANGE_ERROR_IF_FALSE(env, condition, error, message) \
-  do {                                                             \
-    if (!(condition)) {                                            \
-      napi_throw_range_error((env), (error), (message));           \
-      return napi_set_last_error((env), napi_generic_failure);     \
-    }                                                              \
+#define THROW_XERROR_IF_FALSE(errortype, env, condition, error, message)       \
```

Comment:
Should this be `THROW_XERROR_IF_TRUE` here and below?

Reply:
Updated :)